pyo3-derive-backend/src/method.rs

```diff
@@ -29,12 +29,16 @@ pub enum FnType {
     FnCall,
     FnClass,
     FnStatic,
-    PySelf(syn::TypeReference),
+    // self_: &PyCell<Self>,
+    PySelfRef(syn::TypeReference),
+    // self_: PyRef<Self> or PyRefMut<Self>
```

Comment:
Forgotten comments?

Reply:
They are comments for developers.
Rewrote as doc comments.